integrations/terraform-cloud/main.py

```diff
@@ -195,18 +207,8 @@ async def handle_webhook_request(data: dict[str, Any]) -> dict[str, Any]:
 async def on_start() -> None:
     logger.info("Starting Port Ocean Terraform integration")
 
-    if ocean.event_listener_type == "ONCE":
-        logger.info("Skipping webhook creation because the event listener is ONCE")
-        return
-
```

Comment:
You can also add this catch to the part (above) where you are creating the webhook so that it doesn't create it for ONCE event listener type

Reply:
Under the same endpoint (webhook) where verifications will be sent ?